spec/features/login_instructions_spec.rb

```diff
@@ -11,11 +11,19 @@
     login(actor)
   end
 
+  scenario "regular instructions mask password", js: true do
+    query = "password=testpass"
```

Comment:
Eeek I just noticed this QS password thing. Not as horrible as it seems but still a bit dodgy. I'll clean it up while I wait for CI.